docs/featurestore/featurestore.rst

```diff
@@ -829,12 +831,13 @@ Connecting to the Feature Store from a Databricks cluster requires a Feature Sto
 Read access to this API key needs to be given to the AWS role used by the Databricks cluster. 
 Finally, a helper library needs to be installed on the Databricks cluster to connect to the Feature Store.
 
-There exist two helper libraries: hopsworks-cloud-sdk and hops-util-py. 
-Hopsworks-cloud-sdk is easier to set up but does not provide some advanced functionalities present in hops-util-py. If you intend to use hops-util-py follow the instructions in "Setting up roles and API keys" then jump directly to "Installing hops-util-py".
+There exist two helper libraries: hopsworks-cloud-sdk and hops. 
+Hopsworks-cloud-sdk does not require a Spark environment (and is easier to set up) but does not provide some advanced spark-based functionality present in hops. If you intend to use the hops library follow the instructions in "Setting up roles and API keys" then jump directly to "Installing hops library".
 
 **Setting up roles and API keys**
 
-Follow the steps described in `Connecting from Amazon SageMaker`_ for setting up Hopsworks API keys and AWS roles and access to secrets. Ensure to use the role that is specified in the *Advanced Options* when creating a Spark cluster in Databricks.
+You also need to ensure to use the role that is specified in the *Advanced Options* when creating a Spark cluster in Databricks. 
```

Comment:
Having the first sentence of the first paragraph of the section starting wit "also need" seems strange to me. The all sentence seems out of place to me and I'm not sure the user will easily understand what they need to do.

Reply:
line 839 is out of place, will fix.